2_base_and_sensor.ipynb

```diff
@@ -214,28 +396,47 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Let's write a code to move the robot from the initial position to the front of the human.</p>\n",
```

Comment:
"    <p>Let's write some code to move the robot from the initial position to the front of the human.</p>\n",